scripts/scil_split_volume_labels.py

```diff
@@ -0,0 +1,178 @@
+are extracted.
+
+New lookup tables can be added by adding new files to the
+scilpy/data/LUT/ directory.
+"""
+
+
+# Taken from http://stackoverflow.com/a/6512463
+def parseNumList(str_to_parse):
+    m = re.match(r'(\d+)(?:-(\d+))?$', str_to_parse)
+
+    if not m:
+        raise argparse.ArgumentTypeError("'" + str_to_parse + "' is not a " +
+                                         "range of numbers. Expected forms " +
+                                         "like '0-5' or '2'.")
+
+    start = m.group(1)
+    end = m.group(2) or start
+
+    start = int(start, 10)
+    end = int(end, 10)
+
+    if end < start:
+        raise argparse.ArgumentTypeError("Range elements incorrectly " +
+                                         "ordered in '" + str_to_parse + "'.")
+
+    return list(range(start, end+1))
+
+
+def _build_args_parser(luts):
+
+    p = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('label_image',
+                   help='Path of the input label file, '
+                        'in a format supported by Nibabel.')
+    p.add_argument('--output_dir', default='',
+                   help='Put all ouptput images in a specific directory.')
+    p.add_argument('--output_prefix', default='',
+                   help='Prefix to be used for each output image.')
+
+    mode_subparsers = p.add_subparsers(dest='mode',
+                                       help='Output naming schemes.')
+
+    ids_subparser = mode_subparsers.add_parser(
+        'ids', description=IDS_DESCRIPTION,
+        help='Names files using their label id.')
+    ids_subparser.add_argument('-r', '--range', type=parseNumList, nargs='*',
+                               help='Specifies a subset of labels to split, '
+                                    'formatted as 1-3 or 3 4.')
+
+    labels_subparser = mode_subparsers.add_parser(
+        'labels', help='Names the files using a lookup table.')
+    labels_subparser.add_argument(
+        'lut', choices=luts,
+        help='Lookup table, in the file ../data/LUT, '
+             'used to name the output files.')
+
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    # Get the valid LUT choices.
+    path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)),
+        '../data/LUT')
+    luts = [os.path.splitext(f)[0] for f in os.listdir(path)]
+
+    parser = _build_args_parser(luts)
+    args = parser.parse_args()
+
+    required = args.label_image
+    assert_inputs_exist(parser, required)
+
+    label_image = nib.load(args.label_image)
+
+    if not issubclass(label_image.get_data_dtype().type, np.integer):
+        parser.error('The label image does not contain integers. ' +
+                     'Will not process.\nConvert your image to integers ' +
+                     'before calling.')
+
+    label_image_data = label_image.get_data()
```

Comment:
`get_data()` is deprecated now, isn't it? You would need to call get_fdata(), but it supports only float data, so nibabel recommends using the following: `np.asanyarray(img.dataobj)`